x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -679,6 +687,7 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
   ) : (
     <>
       <StepContentWrapper addPadding={!isUpdateView}>
+        <StepDefineRuleNewFeaturesTour />
```

Comment:
Since the step is now bound to the element via an `anchor`, we can put the tour somewhere on the page, for example here. Could be at the whole page level as well.